### DISCUSSION TITLE
2017/03/27/beacon-tower-near-fernley-nv

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#7] [Announcements]
# 2017/03/27/beacon-tower-near-fernley-nv

http://localhost:4000/2017/03/27/beacon-tower-near-fernley-nv

**Comment by jengalas** [comment]
Posted by David Lane
December 12, 2020 at 11:32 pm
_Imported from WordPress_

I have acquired an intact AGA Airway Beacon, nearly identical to the one pictured above. It is serial No. 520. The beacon in the photo is missing its homing beacon which was a circular fresnel type cut glass lens made by Corning glass works. It sat on top of the upper (brass) round housing and was also powered by acetylene gas. I would like to know more about AGA beacon history and would like to find a drawing or photo of one from back in the day thank you.
